norns/reference/parameters.md

```diff
@@ -24,7 +24,7 @@ permalink: /norns/reference/params
 | params:add_binary(id, name, behavior, default)                  | Create a parameter set for either momentary, toggle, or trigger behavior with a default state                                                                                                 |
 | params:print()                                                  | Print the index, name, and value for each parameter to the REPL                                                                                                                               |
 | params:list()                                                   | Print the names of each parameter to the REPL                                                                                                                                                 |
-| params:lookup[id]                                               | Returns the index of a given parameter's string id                                                                                                                                            |
+| params:lookup[id]                                               | Returns the index of a given parameter's string id (note the square brackets                                                                                                                                            |
```

Comment:
rad, ty!
small change request -- the line is missing its closing parenthesis :)